internal/pkg/levee/levee.go

```diff
@@ -44,6 +49,7 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	funcSources := pass.ResultOf[source.Analyzer].(source.ResultType)
 	taggedFields := pass.ResultOf[fieldtags.Analyzer].(fieldtags.ResultType)
 
+	var items []reportItems
```

Comment:
I am not seeing how this in getting updated? Did some merge remove some code?

Reply:
Oh, I meant to convert back to Draft.  It's not anymore.  I was dodging doing too much when resolving merge conflicts.

I also meant to put `TODO` in with the `// build items` comments.  But it's going to require a change to the `reportSource*` methods, which I didn't want to do in one push.

Converting back to Draft